<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>粮食管理</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="pages">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="pageLogin">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="1">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>账号</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbAccount">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>密码</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="editPassword">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnLogin">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>登陆</string>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>西易村福利供应站系统</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageWarehouse">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="3">
         <widget class="QPushButton" name="btnAddFood1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>添加种类</string>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="btnDistribte">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>一键分粮</string>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="QTableView" name="table1"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pagePurchase">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QComboBox" name="cbSupplier2">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>供应商</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbFood2">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>粮食</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnPurchase2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>进货</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTableView" name="table2"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageShipment">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
             <string>卡号</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="editUserId3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>14</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSale3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>出货</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTableView" name="table3">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageAdmin">
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>562</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="btnAddAdmin4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>添加管理员</string>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QTableView" name="table4"/>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btnUpdateAuthPassword">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>修改授权密码</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageUser">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>562</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btnAddUser5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>添加客户</string>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QTableView" name="table5"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageSupplier">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>562</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btnAddSupplier6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>添加供应商</string>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QTableView" name="table6"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionWarehouse"/>
   <addaction name="separator"/>
   <addaction name="actionPurchase"/>
   <addaction name="actionShipment"/>
   <addaction name="separator"/>
   <addaction name="actionAdministrator"/>
   <addaction name="actionCustomer"/>
   <addaction name="actionSupplier"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>资料管理</string>
    </property>
    <addaction name="actionWarehouse"/>
    <addaction name="separator"/>
    <addaction name="actionCustomer"/>
    <addaction name="actionSupplier"/>
    <addaction name="actionAdministrator"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>功能</string>
    </property>
    <addaction name="actionPurchase"/>
    <addaction name="actionShipment"/>
   </widget>
   <addaction name="menuInfo"/>
   <addaction name="menu"/>
  </widget>
  <action name="actionSupplier">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/supplier.png</normaloff>:/images/supplier.png</iconset>
   </property>
   <property name="text">
    <string>供应商</string>
   </property>
   <property name="toolTip">
    <string>供应商信息</string>
   </property>
  </action>
  <action name="actionCustomer">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/user.png</normaloff>:/images/user.png</iconset>
   </property>
   <property name="text">
    <string>客户</string>
   </property>
   <property name="toolTip">
    <string>客户信息</string>
   </property>
  </action>
  <action name="actionWarehouse">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/warehouse.png</normaloff>:/images/warehouse.png</iconset>
   </property>
   <property name="text">
    <string>仓库</string>
   </property>
   <property name="toolTip">
    <string>粮食储备信息</string>
   </property>
  </action>
  <action name="actionAdministrator">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/admin.png</normaloff>:/images/admin.png</iconset>
   </property>
   <property name="text">
    <string>管理员</string>
   </property>
   <property name="toolTip">
    <string>管理员信息</string>
   </property>
  </action>
  <action name="actionPurchase">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/purchase.png</normaloff>:/images/purchase.png</iconset>
   </property>
   <property name="text">
    <string>进货</string>
   </property>
  </action>
  <action name="actionShipment">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/shipment.png</normaloff>:/images/shipment.png</iconset>
   </property>
   <property name="text">
    <string>出货</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
